module.json

```diff
@@ -8,10 +8,9 @@
       "email": "david.bauske@googlemail.com"
     }
   ],
-  "version": "1.10.0",
+  "version": "1.11.01b",
```

Comment:
The module version gets set by the release script. No need to include it in the PR and actually it will make the release script fail. Please revert it :)